doc/classes/ColorRect.xml

```diff
@@ -38,6 +38,10 @@
 	</methods>
 	<members>
 		<member name="color" type="Color" setter="set_frame_color" getter="get_frame_color">
+			The color to fill the [code]ColorRect[/code].
+			[codeblock]
+			$ColorRect.color = Color(1, 0, 0, 1) # Set ColorRect node's color to red
```

Comment:
Im not sure if this kind of examples are useful at all, i mean anyone who nows about gdscript, and about the color class can clearly infer this.

Reply:
Can help for new users I guess

Reply:
Tough call. There's been many requests for more code examples in the classref. There are a lot of beginners in the community, so it's hard to know when to and when not to include examples. For this one I find it very basic, like @djrm suggests